src/main/java/uk/gov/hmcts/reform/sscs/controller/SyaController.java

```diff
@@ -36,8 +36,9 @@
     public ResponseEntity<String> createAppeals(@RequestBody SyaCaseWrapper syaCaseWrapper) {
         log.info("Appeal with Nino - {} and benefit type {} received", syaCaseWrapper.getAppellant().getNino(),
                 syaCaseWrapper.getBenefitType().getCode());
-        submitAppealService.submitAppeal(syaCaseWrapper);
-        log.info("Appeal with Nino - {} and benefit type - {} processed successfully",
+        Long caseId = submitAppealService.submitAppeal(syaCaseWrapper);
```

Comment:
It's better to use primitives rather than boxed primities, since you are not creating new objects each time.

Reply:
Yes but here its returning the caseId which is boxed primitives Long